string connection_name
string device_iface
---
string message